src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -463,17 +463,24 @@ export default class Aircraft {
     }
 
     /**
+     * Returns a true value if there is a match from the callsignToMatch
+     *
      * @for AircraftInstanceModel
      * @method matchCallsign
      * @param callsign {string}
      */
-    matchCallsign(callsignToMatch) {
-        if (callsignToMatch === '*') {
-            return true;
-        }
-
-        return _isEqual(callsignToMatch.toUpperCase(), this.getCallsign());
-    }
+     matchCallsign(callsignToMatch) {
+         if (callsignToMatch === '*') {
+             return true;
+         }
+         // checks to see if the given call sign matches the airline Id + callsign format
+         if (_isEqual(callsignToMatch.toUpperCase(), this.getCallsign())){
```

Comment:
I'd like to see the first if in this method removed.  What you could do instead is:
```javascript
const shouldMatchAnyCallsign = callsignToMatch === '*';
```

then in this if you can do:
```javascript
if (shouldMatchAnyCallsign || _isEqual(callsignToMatch.toUpperCase(), this.getCallsign())) { }
```